tests/testcases/Group10-Vulnerability/10-06-Change-scan-configuration.md

```diff
@@ -0,0 +1,31 @@
+Test 10-06 Change scan configuration
+=======
+   
+# Purpose:   
+To verify scan configuration can be changed correctly.   
+   
+# References:   
+User guide   
+   
+# Environment:   
+* This test requires that a Harbor instance is running and available.   
+* Harbor is installed with clair enable.   
+* A linux host with Docker CLI installed.   
+   
+# Test Step   
+   
+1. Login harbor as admin.   
+2. Push a image to a project.   
+3. Click configuration.   
+4. Click vunlerablity.   
+5. Set scan all to none.   
+6. Click scan now.   
+7. Check the image scan time.   
+   
+# Expected Outcome:   
+* Step 4 if a new installed Harbor,vulnerabilit not ready hit should exist.  
+* Step 5 schedule time input should disappear.  
```

Comment:
Should also verify the scan now is disabled, right?

Also after that period of time, he should be able to trigger "scan now" again.